src/content/general/management-clusters/index.md

```diff
@@ -21,13 +21,19 @@ As we are fully convinced of Kubernetes as a platform for building platforms, we
 
 As well as operators, our management clusters run supporting components such as our monitoring stack and our [web UI]({{< relref "/ui-api/web" >}}). All components are deployed using the Giant Swarm [App Platform]({{< relref "/app-platform/overview" >}}).
 
+Users have direct access to the Kubernetes API of the management cluster. The gives full access to the automation we've built into our operators, as well as visibility into the components running within the management cluster. Documentation can be found [here]({{< relref "/ui-api/management-api/" >}}).
+
+We do not support the upstream Cluster API for all our providers yet, but are fully committed to supporting this open standard soon.
+
 ## Workload Cluster releases
 
 Our workload clusters are versioned using [workload cluster releases]({{< relref "/general/releases" >}}). From a workload cluster point of view, upgrades are described in [cluster upgrades]({{< relref "/general/cluster-upgrades" >}}).
 
-When we publish a new workload cluster release, we create a new [Release]({{< relref "/ui-api/management-api/crd/releases.release.giantswarm.io.md" >}}) custom resource and any new operator versions are deployed to the management cluster.
+A core focus of our development in our versioning is to guarantee stability and reliability. Our upgrades follow a lockstep approach, where we move from one tested set of versions of our components to the next set of versions of our components. This strict versioning also allows you to test out new versions, test upgrades, and even integrate cluster creation into your own CI, such as to run integration tests.
```

Comment:
```suggestion
A core focus of development in our versioning approach is to guarantee stability and reliability. Our upgrades follow a lockstep approach, where we move from one tested set of versions of our components to the next set of versions of our components. This strict versioning also allows you to test out new versions, test upgrades, and even integrate cluster creation into your own CI, such as to run integration tests.
```

The two uses of our doesn't flow for me for some reason. What about this? I think our approach is something unique about our offer.

So overall I like this explanation a lot and good to go deeper here. 👍